### DISCUSSION TITLE
Where can I apply for the GitHub university 2022?

### DISCUSSION THREAD

**Discussion by HackerstoyanovBG** [community/community#35930] [Other Feature Feedback, Questions, & Ideas] [answered]
Where can I apply for the GitHub university 2022?
Also someone told me I have to register with company or something, does the company I work in counts?

**Comment by MitkoVtori** [comment, marked as answer]
Hello, you can register from here: https://githubuniverse.com/?utm_source=github&utm_medium=product&utm_campaign=2022-product-WW-Universe&utm_content=save-the-date-promo
You also can register with our company.

**Reply by HackerstoyanovBG** [reply]
Bruh, didn't expected to see you here **boss** 😛
Such a long link, try better next time XD (pls don't fire me, I want to be on this event) **xd**
